packages/flutter_tools/lib/src/ios/ios_emulators.dart

```diff
@@ -16,17 +16,26 @@ class IOSEmulators extends EmulatorDiscovery {
   bool get canListAnything => globals.iosWorkflow.canListEmulators;
 
   @override
-  Future<List<Emulator>> get emulators async => getEmulators();
+  Future<List<Emulator>> get emulators async {
+    final List<IOSSimulator> simulators = await globals.iosSimulatorUtils.getAvailableDevices();
```

Comment:
should this be cached somewhere?

Reply:
It's only called once, and that output is used to create the `IOSSimulator` which is then passed around. The underlying command runs extremely fast compared to the real-device equivalent, it's not doing any fancy mux or networking tricks, it returns practically instantly.  If we called it more than twice the caching would probably be worth it though.